src/main/java/org/kuali/kfs/kim/document/IdentityManagementPersonDocument.java

```diff
@@ -482,8 +484,19 @@ public void doRouteStatusChange(final DocumentRouteStatusChange statusChangeEven
         if (getDocumentHeader().getWorkflowDocument().isProcessed()) {
             setIfGroupsEditable();
             setIfRolesEditable();
+
+            for (final PersonDocumentRole role : roles) {
```

Comment:
Since this is a modification to an overlayed file, I would suggest adding a comment to mark this spot as a customized one.